src/iologindata.cpp

```diff
@@ -613,22 +614,21 @@ bool IOLoginData::saveItems(const Player* player, const ItemBlockList& itemList,
 		}
 
 		if (Container* container = item->getContainer()) {
-			queue.emplace_back(container, runningId);
+			containers.emplace_back(container, runningId);
 		}
 	}
 
-	while (!queue.empty()) {
-		const ContainerBlock& cb = queue.front();
+	for (size_t i = 0; i < containers.size(); i++) {
+		const ContainerBlock& cb = containers[i];
```

Comment:
```suggestion
	for (const ContainerBlock& cb : containers) {
```

Reply:
That wouldn't work because we append to the containers vector in this loop.